src/Tribe/RSVP.php

```diff
@@ -2273,33 +2299,41 @@ public function generate_tickets_for( $product_id, $ticket_qty, $attendee_detail
 	public function parse_attendee_details() {
 		$order_id = self::generate_order_id();
 
-		$attendee_email     = empty( $_POST['attendee']['email'] ) ? null : sanitize_email( $_POST['attendee']['email'] );
+		$first_attendee = [];
+
+		if ( isset( $_POST['tribe_ticket_attendee'] ) && ! empty( $_POST['tribe_ticket_attendee'] ) ) {
+			$first_attendee = $_POST['tribe_ticket_attendee'][0];
```

Comment:
* :warning: Detected usage of a non-sanitized input variable: $_POST['tribe_ticket_attendee'][0]
* :no_entry_sign: Processing form data without nonce verification.
* :no_entry_sign: Detected usage of a possibly undefined superglobal array index: $_POST['tribe_ticket_attendee'][0]. Use isset() or empty() to check the index exists before using it

<sub>hash: feca720489537a165f3c7c540b074b76</sub>